### PR TITLE
1.0 docs structure

### PR DESCRIPTION
**Pull request by jamilbk**
Refs #2021

### CODE REVIEW

File: website/src/app/kb/quickstart/readme.mdx [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Quickstart Guide
+
+{/* TODO */}
```
Comment (jamilbk) [inline website/src/app/kb/quickstart/readme.mdx]: @bmanifold Can probably just copy-paste the Quickstart guide draft here.

Comment (bmanifold) [reply website/src/app/kb/quickstart/readme.mdx]: Just pushed a commit to this branch with the Quickstart guide content.  If you'd rather it be in a separate PR, feel free to reset and force push over it.

File: website/src/app/kb/quickstart/readme.mdx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  height={50}
+  className="mx-auto shadow rounded"
+/>
+
+
+## Prerequisites
+
+* Firezone account (Don't have an account? Get one here https://app.firezone.dev/sign_up)
+* Resource you want to give users secure access to (e.g. prod server, database, SaaS application, subnet, etc...)
+* Server or VM (with the ability to deploy a docker container or Linux application) that can connect to the resource, and the internet
```
Comment (jefferenced) [inline website/src/app/kb/quickstart/readme.mdx]: would "to both the resource, and the internet" be more clear here?

**Review by jefferenced — APPROVED** [review]
looks good!

File: website/src/app/kb/quickstart/readme.mdx (hunk 2 above)
Comment (bmanifold) [reply website/src/app/kb/quickstart/readme.mdx]: Sounds good to me 👍 
I'll get it updated.

File: website/src/app/kb/quickstart/readme.mdx (hunk 1 above)
Comment (jamilbk) [reply website/src/app/kb/quickstart/readme.mdx]: 👍

**Pull request merged**